README.md

```diff
@@ -39,88 +39,79 @@ To get in touch with us, please join our [discord server](https://discord.gg/pGZ
 
 ## Quickstart
 
-Follow this guide to learn how to quickly set up Arch and integrate it into your generative AI applications.
+Follow this quickstart guide to use arch gateway to manage access keys, provide unified access to upstream LLMs and to provide e2e observability. Later we will see how you can use arch gateway to build a Gen AI application.
```

Comment:
We need to highlight the most differentiating feature first - build apps with Arch using simple APIs. Followed by observability, access key management and other features. We are uniquely positioned on and shouldn't bury the lede: https://www.merriam-webster.com/wordplay/bury-the-lede-versus-lead